expan/core/results.py

```diff
@@ -4,40 +4,49 @@
 import numpy as np
 import pandas as pd
 import version
+import statistics as statx
+from scipy.stats import norm
 
 # from tests.tests_core.test_data import generate_random_data
 
 from debugging import Dbg
+from pdb import set_trace
+
+mandatory_index_levels = [
+	'metric',
+	'subgroup_metric',
+	'subgroup',
+	'statistic',
+	'pctile']
 
```

Comment:
I had to use this later on (line 334) in an external function, maybe we can make this array always global?